src/core/Akka.Remote/Transport/AkkaPduCodec.cs

```diff
@@ -365,10 +367,15 @@ public override ByteString ConstructDisassociate(DisassociateInfo reason)
         /// <returns>TBD</returns>
         public override ByteString ConstructHeartbeat()
         {
-            return ConstructControlMessagePdu(CommandType.HEARTBEAT);
+            return ConstructControlMessagePdu(CommandType.Heartbeat);
         }
 
         /// <summary>
+        /// Indicated RemoteEnvelope.Seq is not defined (order is irrelevant)
+        /// </summary>
+        private const ulong SeqUndefined = ulong.MaxValue;
```

Comment:
A kind of. I didn't want to another field indicating Seq should be ignored. The approach suggested to simulate optional fields will add field too. So it was an attempt to leverage efforts and result. I agree it's smelling.